### PR TITLE
Create Category Model

### PR DESCRIPTION
**Pull request by michaelwehrley**
Create Category Model

* Category attributes exist.
* Add in Rails Admin.
* Update rspec gem and .rspec.
* Added in authentication to access this model via rails_admin.

[Finishes: #91548060]
[Finishes: #91531724]

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Are sets of chapters that display text, image, and an active
+# state to the user so they know what module they are in.
+class Category < ActiveRecord::Base
```
Comment (ericcf) [inline app/models/category.rb]: If the table for this model has `null: false` for columns, should the model have corresponding validations?

Comment (michaelwehrley) [inline app/models/category.rb]: It will, but that model doesn't yet exist - in a different pull request.

Comment (ericcf) [inline app/models/category.rb]: I meant for example: should Category have presence validations for the `title` and `media_src` attributes?

Comment (michaelwehrley) [inline app/models/category.rb]: Yeah, those should be in this pull request.

**Pull request closed without merging**